.vscode/launch.json

```diff
@@ -102,12 +102,11 @@
 			"type": "node",
 			"request": "launch",
 			"name": "Attach Web Test",
-			"program": "${workspaceFolder}/node_modules/vscode-test-web/out/index.js",
+			"program": "${workspaceFolder}/node_modules/@vscode/test-web/out/server/index.js",
```

Comment:
I was trying to upgrade packages in https://github.com/microsoft/vscode-pull-request-github/pull/7305 but I forgot to update the the launch.json